config/auth-log.php

```diff
@@ -11,11 +11,14 @@
         Illuminate\Auth\Events\PasswordReset::class => 'handleAuthenticatablePasswordReset',
     ],
     'credentialsToOmit' => [
-        'password',
+        // 'password',
```

Comment:
I would not recommend this - for security the default should be to omit any password details